File: answers/AkhandAgrawal/Conditionals/question 1.py
basic = int(input("Enter the salary: "))
grade = input("Enter the grade(in uppercase): ")
if grade == 'A':
    allow = 1700
elif grade == 'B':
    allow = 1500
else:
    allow = 1300
hra = (20/100) * basic
da = (50/100)*basic
pf = (11/100) * basic
totalSalary = int(basic + hra + da + allow-pf)
print(totalSalary)

File: answers/tounovdattaTXD/Conditionals/question1.c
#include <stdio.h>

int main()
{
    int basic, allow, totalsalary;
    char grade;
    float hra, da, pf;
    printf("Enter your basic salary and your grade(in uppercase):\n");
    scanf("%d %c", &basic, &grade);
    switch(grade)
    {
        case 'A': allow= 1700; 
        break;
        case 'B': allow= 1500;
        break;
        default: allow= 1300;
        break;
    }
    hra= (20*basic)/100;
    da= (50*basic)/100;
    pf= (11*basic)/100;
    totalsalary= basic+hra+da+allow-pf;
    printf("The total salary is: %d", totalsalary);

    return 0;
}
